core/templates/dev/head/css/oppia.css

```diff
@@ -870,6 +870,50 @@ textarea {
   border: none;
 }
 
+/*Prevent Exploration Player From collapse On Small Screen*/
```

Comment:
@seanlip i have shifted change from logo to icon to exploration player, it doesn't show up on other pages.